src/components/tooltip/NetworkMarker.js

```diff
@@ -351,7 +351,15 @@ export class TooltipNetworkMarker extends React.PureComponent<Props> {
 
   render() {
     const { payload } = this.props;
-    const mimeType = guessMimeTypeFromNetworkMarker(payload);
+    let mimeType = payload.contentType;
```

Comment:
If the value is `null` or the empty string here, we'll just not show anything in the tooltip.
We could, if we wanted to, do something to to disambiguate these two cases. If the Gecko profiler patch lands as it is now, `null` means no HTTP response request was received and so there was nothing to read a content type from. The empty string means the request was received, but it didn't have any content type in its header.

Reply:
I think that's the good call, however the difference you're outlining should be in a comment somewhere, problably in the type definition :-)

Reply:
I'll add it :-)